clm_demeter/prepare_baselayer.py

```diff
@@ -0,0 +1,293 @@
+import pandas as pd
+import numpy as np
+import matplotlib.pyplot as plt # to generate plots
+from netCDF4 import Dataset     # to work with NetCDF files
+import os                       # operating system interface
```

Comment:
You have a commented line in the `read_baselayer` method that uses `os`, otherwise it is not used.  If the commented line need be removed, remove the import of `os` as well.